uiowa/uiowa
# [Event feature] Attendance Mode field addition

Resolves #5940 
Resolves #6463 

# Updated tests

`ddev blt ds --site=iowasummerwritingfestival.uiowa.edu`
Check that it has a clean sync and update

On any site with the event feature:
Edit existing or create new events
Set attendance mode to hybrid and add both a location and a virtual event info; save
Edit, and change attendance mode to in-person and save; see that the virtual event info has been removed
Repeat above, but instead change attendance mode to online and save; see that the location info has been removed





# How to test


`git checkout event-attendance-mode && git pull && ddev blt ds --site=default && ddev drush @default.local config-split:activate event && ddev drush @default.local uli`
Add an event using the new Attendance mode field
Check that if the attendance mode:

- is "in person" the location field shows but not the virtual
- is "online" the virtual shows but not the location
- is "hybrid," both virtual and location fields show

Check that the teasers and node display do not show the attendance mode
View the source, and check that the attendance mode is appearing in the meta markup

Test the sites that have the split activated, and that the update hook runs without errors
Check out existing events and see that they have an appropriate attendance mode set (see https://github.com/uiowa/uiowa/pull/6464#issuecomment-1557395672 for added context as of 05/22)
Check the same as on default--the values don't display directly, but update the meta markup
`ddev blt ds --site=iowasummerwritingfestival.uiowa.edu`
`ddev blt ds --site=admissions.uiowa.edu`
`ddev blt ds --site=oniowa.uiowa.edu`

Weigh in on https://github.com/uiowa/uiowa/pull/6464#issuecomment-1557542989 and if update hook should move to `sitenow_events` or stay in the profile.
